client/snippets/snippets.json

```diff
@@ -39,5 +52,267 @@
+        "description": "Verb"
+    },
+    "For Each with Assembly ID": {
+        "prefix": "\\foreach",
+        "body": [
+            "\\foreach(${1:var}, ${2:in}, ${3:id})",
+            "\t$0",
+            "\\endForeach"
+        ],
+        "description": "“For each” loop with assembly ID"
+    },
+    "For Each": {
+        "prefix": "\\foreach",
+        "body": [
+            "\\foreach(${1:var}, ${2:in})",
+            "\t$0",
+            "\\endForeach"
+        ],
+        "description": "“For each” loop"
+    },
+    "Static List": {
+        "prefix": "\\beginList",
+        "body": [
+            "\\beginList",
+            "\t\\nextItem",
+            "\t\t${1:xxx}",
+            "\t",
+            "\t\\nextItem",
+            "\t\t${2:yyy}",
+            "\t$0",
+            "\\endList"
+        ],
+        "description": "Static List"
+    },
+    "Static List Item": {
+        "prefix": "\\nextItem",
+        "body": [
+            "\\nextItem",
+            "\t$0"
+        ],
+        "description": "An item in a static list"
+    },
+    "Synonyms": {
+        "prefix": "\\beginSynonym",
+        "body": [
+            "\\beginSynonym",
+            "\t\\choice",
+            "\t\t${1:xxx}",
+            "\t",
+            "\t\\choice",
+            "\t\t${2:yyy}",
+            "\t$0",
+            "\\endSynonym"
+        ],
+        "description": "List of alternatives"
+    },
+    "Synonyms (shorthand)": {
+        "prefix": "synonyms",
+        "body": [
+            "\\beginSynonym",
+            "\t\\choice",
+            "\t\t${1:xxx}",
+            "\t",
+            "\t\\choice",
+            "\t\t${2:yyy}",
+            "\t$0",
+            "\\endSynonym"
+        ],
+        "description": "List of alternatives"
+    },
+    "Synonym Choice": {
```

Comment:
I would just copy-paste my code. Not needed, I think.

Reply:
* It's small.
* I tried using it and found the automatic newline + indentation very pleasant.
* Most people are very inefficient when trying to copy and paste stuff (the abusively use the mouse and don't realize that in order to place the cursor right after your selection when you've just copied it you can simply paste it on itself).